binding/Binding/VkTypes.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace SkiaSharp
+{
+	public struct VkHandle
+	{
+		public IntPtr Handle;
+	}
+
+	public struct VkNonDispatchableHandle
+	{
+		public ulong Handle;
```

Comment:
Should this be a `UIntPtr`? But again, are we using this directly, or do we just need the handle and not the struct? I am not sure if we want to define vulkan types in skiasharp

Reply:
https://vulkan.lunarg.com/doc/view/1.0.33.0/linux/vkspec.chunked/ch02s03.html

`Non-dispatchable handle types are a 64-bit integer type whose meaning is implementation-dependent, and may encode object information directly in the handle rather than pointing to a software structure.`

Reply:
Sounds like `ulong` to me, why `UIntPtr`?